Fix horizontal bar sizing

SparrowBrain opened the pull request:
Horizontal bars now will automatically size depending on the window size.

This is implemented using multibinding, and calculating the width of the bar depending on:
* MaxWidth (set via codebehind when grid size changes)
* Percentage (set in ViewModel depending on the playtime value)
* Offset (set in view). This is used to leave space for time played text on the right.

fixes #7

JonasPTFL reviewed (approved):
I dont know if that is intentional, but when i resize the window below a specific width, the horizontal bars completly disappear (see images below).

Width 1:
![grafik](https://github.com/user-attachments/assets/0314bb7f-2e4b-49ad-95c1-7a949f5c6ec6)

Width 2:
![grafik](https://github.com/user-attachments/assets/6cc36a71-c388-4a44-b64e-95a183fd1a8f)

Apart from that, the changes look good to me, and everything seems to be working fine.
I’ve approved the PR. If you want to make any updates, feel free to push your changes and tag me for another review. Otherwise, just merge the PR.

SparrowBrain commented:
@JonasPTFL Good catch with disappearing bars. It was jsut math - at some point you have less than 1 pixel to show and it would disappear. I've changed it now to always at least have one pixel.

Additionally, I've added a breakpoint - now the report will take the full width of the view, when the view is less than 1000px.

JonasPTFL reviewed (approved):
Horizontal bars do not disappear now. Looks really good!
